apps/www/_blog/2023-04-07-launch-week-7-hackathon.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: 'Launch Week 7 Supa AI Hackathon'
+description: Build an Open Source Project, Win $1500 and limited edition Supa AI swag packs!
+author: ant_wilson
+image: lw7-hackathon/thumbnail.jpg
+thumb: lw7-hackathon/thumbnail.jpg
```

Comment:
Waiting on the design team

Reply:
can we do all this in notion?

Reply:
We can. It's in the LW7 board!
https://www.notion.so/supabase/LW7-Iota-search-Horsey-50ad028d176341b882348300666c246a?p=e6ef8462f5a1460ba74e3d563fce6f23&pm=s